src/sentry/static/sentry/app/utils/discover/genericDiscoverQuery.tsx

```diff
@@ -42,6 +42,10 @@ export type DiscoverQueryProps = {
    * A callback to set an error so that the error can be rendered in parent components
    */
   setError?: (msg: string | undefined) => void;
+  /**
+   * Sets referrer parameter in the API Payload.
```

Comment:
Can we add a comment referring to the allow list so someone looking at this is aware they can't just use any referrer they want which is what the type here is suggesting.